fiftyone/core/expression_ast.py

```
@@ -0,0 +1,644 @@
+            **{k: _decode(v) for k, v in node.get("kwargs", {}).items()},
+        )
+
+    if t == "call":
+        target = _decode(node["self"])
+        method = getattr(target, node["op"])
+        return method(
+            *[_decode(n) for n in node["args"]],
+            **{k: _decode(v) for k, v in node.get("kwargs", {}).items()},
+        )
+
+    raise ValueError("unknown view expression node type %r" % t)
```

Ruff (0.15.21)

[warning] Use format specifiers instead of percent format

Replace with format specifiers

(UP031)